### PR TITLE
Deeper spotbugs checks

### PR DESCRIPTION
**Pull request by pushker001**
Increase the efficiency of checking spots-bug

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
   <properties>
     <jenkins.version>2.361.4</jenkins.version>
+     <spotbugs.effort>Max</spotbugs.effort>
+     <spotbugs.failOnError>true</spotbugs.failOnError>
```
Comment (MarkEWaite) [inline pom.xml:20]: Not needed, since this is the default.
```suggestion
```

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
Thanks.  Please remove the `spotbugs.failOnError` setting since it is a duplicate of the default setting

**Comment by pushker001** [comment]
@MarkEWaite I want to write a contributing guide for this plugin can you provide me some material to write in contributing guide

**Comment by pushker001** [comment]
@MarkEWaite wants to write the contributing guide for this plugin can you provide some sources to write the contributing guide

**Comment by MarkEWaite** [comment]
> @MarkEWaite wants to write the contributing guide for this plugin can you provide some sources to write the contributing guide

No, I won't provide more sources than are already provided in https://www.jenkins.io/doc/developer/tutorial-improve/add-a-contributing-guide/

**Pull request merged**